test/cypress/features/transaction/transaction.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable */
+import { Given, Then } from 'cypress-cucumber-preprocessor/steps';
+import { ss } from '../../../constants';
+
+Given(/^Given I am on Transactions page$/, function () {
+	cy.visit('/transactions');
+})
+
+Then(/^I should see more than ([^\s]+) transactions in table$/, function (trnxNumber) {
+  cy.get(ss.transactionRow).should('have.length.greaterThan', Number(trnxNumber));
+});
```

Comment:
there already exists a step definition for this too in `common.js` also since this test can be run against devnet instead of using greaterThan, a defined length can be used since the length of the transaction to be displayed is always known.

Reply:
@eniolam1000752 While I was aware of using exact length, there are cases when such user might make further transactions in the future so using an exact length would work for the time being but break tests in the future. In such case, since the minimum number of transactions is known at the time of writing the test, then expecting such number or more can future proof the test especially in a case where the expected result is less than 20 for the time being